vizro-core/docs/pages/user_guides/layouts.md

```diff
@@ -46,11 +46,11 @@ will automatically be stacked underneath each other. If that is your desired lay
 
 
 
-## Customizing the grid arrangement
+## Configuring the grid
```

Comment:
I know it's annoying since I already approved the PR and would be good to get it merged... But I re-read this page and something I think is missing is a quick explanation of horizontal stacking. Currently we say
>  If no layout is specified, all charts/components will automatically be stacked underneath each other.

We should say also:
* this corresponds to `grid = [[0], [1], [2]]` because this specifies 3 equal rows, each of which contains a single component that takes up the whole width
* you can achieve stacking in columns with `grid = [[0, 1, 2]]` because this specifies 1 row that takes up the whole width, then divided into 3 equal columns

Maybe a short new section right above "Basic Example" called "stacking" or something would explain this?

Reply:
That makes sense! 💡 

![Screenshot 2024-01-30 at 19 13 29](https://github.com/mckinsey/vizro/assets/90609403/8114f396-f15e-42c6-84c6-9844523ee07b)